### ISSUE TITLE
Please clarify licence

### ISSUE THREAD

**Issue by andybak** [Battlehub0x/Unity_RuntimeEditor#1]
I notice this is a paid asset. Unfortunately many people will assume that it's presence on Github implies that it's open source in the traditional sense. If this isn't what you intend then it would be useful to add a licence or copyright message making this clear.

**Comment by Battlehub0x** [comment]
Hi, currently https://choosealicense.com/no-permission/
In future I will probably will licence it under MIT.

**Comment by andybak** [comment]
> Hi, currently https://choosealicense.com/no-permission/

Yeah. That's what I assumed but other people might not. A quote from that page:

> _You don’t have to do anything to not offer a license. You may however wish to add a copyright notice and statement that you are not offering any license in a prominent place (e.g., your project’s README) so that users don’t assume you made an oversight._


> In future I will probably will licence it under MIT.

That would be incredibly generous of you.

**Comment by hvallee** [comment]
Hello @Battlehub0x  There is a branch with an MIT license. Does it mean only this branch can be used as MIT and the other are https://choosealicense.com/no-permission/ ?
Thank you for your answer.
